extension/src/popup/components/WarningMessages/index.tsx

```diff
@@ -1126,18 +1126,27 @@ export const BlockAidSiteScanLabel = ({
 
 export const BlockaidTxScanLabel = ({
   scanResult,
+  isPopup = false,
 }: {
   scanResult: BlockAidScanTxResult;
+  isPopup?: boolean;
 }) => {
   const { t } = useTranslation();
   const { simulation, validation } = scanResult;
 
   if (simulation && "error" in simulation) {
+    const header = t("This transaction is expected to fail");
+    if (isPopup) {
```

Comment:
the main change here is to re-use this logic for showing sim and validation errors. In the send/swap flow, we just show a different a component